hydroflow_lang/src/graph/serde_graph.rs

```diff
@@ -39,9 +44,11 @@ impl SerdeGraph {
 
         fn write_mermaid_prelude(write: &mut impl std::fmt::Write) -> std::fmt::Result {
             // intro
+            #[allow(clippy::write_literal)]
```

Comment:
i don't understand why it's better to write this way?

Reply:
changed it back, not a big difference, just whether the `{{` are escaped or not